src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts

```diff
@@ -343,7 +344,10 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
       .subscribe((res) => {
         const _parsedResponse = this.courseProgressService.getContentProgressState(req, res);
         this.progressToDisplay = Math.floor((_parsedResponse.completedCount / this.courseHierarchy.leafNodesCount) * 100);
-        this.contentStatus = _parsedResponse.content || [];
+        this.primaryCategoryType = (_.get(this.courseHierarchy, 'primaryCategory')) === 'PIAA Assessment';
```

Comment:
@sohailamjad12 is this change to not display Best Score or assessment Progress ?